tfjs-backend-webgl/src/conv_packed_gpu_depthwise.ts

```diff
@@ -193,18 +193,21 @@ export class DepthwiseConvPacked2DProgram implements GPGPUProgram {
               if (dilationWidth > 1) {
                 mainLoop += `
                     xCOffset -= 2;
-                    if (xCOffset >= 0 && xCOffset < inDims[1] && xTexelC${
-                    colIndex}Ready == 0) {
-                      xTexelC${colIndex} = getX(batch, xR, xCOffset, d1);
-                      xTexelC${colIndex}Ready = 1;
+                    if (xCOffset >= 0 && xCOffset < inDims[1]) {
+                     previous = getX(batch, xR, xCOffset, d1);
```

Comment:
In this case, it does `xCOffset -= 2` and `getX(batch, xR, xCOffset, d1)`, which means it want to read the texel right before `xTexelC${colIndex + 1}`, which is not necessarily `xTexelC${colIndex}` (for example, dilation=3).